hikari/impl/stateful_event_manager.py

```diff
@@ -152,11 +152,21 @@ async def on_guild_create(self, shard: gateway_shard.GatewayShard, payload: data
         for presence in event.presences.values():
             self._cache.set_presence(presence)
 
+        self._cache.clear_voice_states_for_guild(event.guild.id)
         for voice_state in event.voice_states.values():
             self._cache.set_voice_state(voice_state)
 
-        if event.guild.is_large and (self._intents is None or self._intents & intents_.Intents.GUILD_MEMBERS):
-            await self._app.chunker.request_guild_chunk(event.guild)
+        members_declared: int = True
+        presences_declared: int = True
+        if self._intents is not None:
+            members_declared = self._intents & intents_.Intents.GUILD_MEMBERS
+            presences_declared = self._intents & intents_.Intents.GUILD_PRESENCES
+
+        # When intents are enabled discord will only send other member objects on the guild create
+        # payload if presence intents are also declared, so if this isn't the case then we also want
```

Comment:
Is this a bug or "a feature"?

Seems like a dumb feature, if it is the latter. Perhaps they need to document this clearly?

Reply:
It's a known feature